examples/PointCloud/visualize_pointcloud.py

```diff
@@ -1,9 +1,13 @@
 import depthai as dai
-import open3d as o3d
 from time import sleep
 import numpy as np
 import cv2
 import time
+try:
+    import open3d as o3d
+except ImportError:
+    print("Open3D not installed, to be able to run the example install Open3D with: python3 -m pip install open3d")
```

Comment:
Rather use `sys.executable` instead python3.
Also raise instead exit, or raise a new exception with this message

Reply:
```
import sys

try:
    import open3d as o3d
except ImportError:
    sys.exit("Critical dependency missing: Open3D. Please install it using the command: '{} -m pip install open3d' and then rerun the script.".format(sys.executable))

```

Reply:
Agree on `sys.executable`.

Why raise instead of error though? If we catch exactly what the error is - isn't it cleaner to output a clean "what to do" message than what's the output of an uncatched error:

```
matevz@matevz-luxmmPC:~/Luxonis/Depthai/depthai-python/examples/PointCloud$ python visualize_pointcloud.py 
Open3D not installed, to be able to run the example install Open3D with: /usr/bin/python -m pip install open3d
Traceback (most recent call last):
  File "visualize_pointcloud.py", line 8, in <module>
    import open3d as o3d
ModuleNotFoundError: No module named 'open3d'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "visualize_pointcloud.py", line 12, in <module>
    raise ImportError(errorMessage)
ImportError: Open3D not installed, to be able to run the example install Open3D with: /usr/bin/python -m pip install open3d
```